pkg_defs/ska3-core/meta.yaml

```diff
@@ -197,7 +197,7 @@ requirements:
     - pyrsistent ==0.16.0
     - pysocks ==1.7.1
     - pytables ==3.6.1
-    - pytest ==5.4.3
+    - pytest ==6.2
```

Comment:
The ska3-flight meta still references 2020.14 for core. We definitely want to check that all our tests are still working with 6.2.

Reply:
Fixed